client/src/utilities/api-calls.js

```diff
@@ -26,4 +26,21 @@ const addNewEntry = (newEntry) => {
   .catch(error => console.log(error));
 }
 
-export { getAllEntries, addNewEntry };
\ No newline at end of file
+const updateEntry = (updatedEntry, id) => {
+  return fetch(`http://localhost:3001/api/v1/entries/${id}`, {
+    method: 'PUT',
+    headers: { 'Content-Type': 'application/json' },
+    body: JSON.stringify({journal_entry: updatedEntry})
+  })
+  .then(response => {
+    if (response.ok) {
+      console.log('RESPONSE', response)
```

Comment:
Super small thing but wanted to point out this leftover console.log

Reply:
THANK YOU